doc/src/devdocs/compiler.md

```diff
@@ -18,7 +18,7 @@ ensuring they are valid both for the current runtime and after deserialization.
 
 When emitted into the object file, these globals are stored as references
 in a large `gvals` table. This allows the deserializer to reference them by index,
-and implement a custom manual GOT-like mechanism to restore them.
+and implement a custom manual Global Offset Table (GOT) mechanism to restore them.
```

Comment:
that's not very meaningful - if GOT means something specific, in what way is our thing related and in what way is it different?